utils/scripts/install-el8.sh

```diff
@@ -54,6 +54,7 @@ dnf --nodocs install \
     patchelf \
     python3-devel \
     python3-pip \
+    python3-pyxattr \
```

Comment:
This should be installed through pip as it's listed in https://github.com/daos-stack/daos/blob/master/requirements.txt#L8

Reply:
We only want to use Pypi installed modules where we have to in CI.
For internal CI, we should have all Python modules available as RPMs.

Reply:
No we don't, we use Pypi for all python where we can to get newer versions and consistency across distros.

Reply:
Pypi modules can not be used for package building.   Which means our unit testing with builds using Pypi modules is not going to match our functional testing or what is going to be delivered in packages.

Reply:
Unit test builds are completely different to functional testing.  Functional testing as you say uses rpm packages.